index.html

```diff
@@ -1805,7 +1805,7 @@ <h2>The Ed25519Signature2020 Suite</h2>
       <p>
         `Ed25519Signature2020` is an earlier version of a cryptographic suite
         for the usage of the EdDSA algorithm and Curve25519. While it has
-        been used in production systems, new implementations should use <a href="#eddsa-rdfc-2022">`edssa-2022`</a>
+        been used in production systems, new implementations should use <a href="#eddsa-rdfc-2022">`edssa-rdfc-2022`</a>
```

Comment:
Should be `eddsa-rdfc-2022` not `edssa-rdfc-2022`

Reply:
We will get there, eventually 😀

(changed it)